src/styl/fonts/fonts.styl

```diff
@@ -0,0 +1,89 @@
+
+// Druk
+
+@font-face {
+	font-family: 'Druk';
+	src: url('../static/fonts/Druk-Bold.eot');
+	src: url('../static/fonts/Druk-Bold.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/Druk-Bold.woff') format('woff'),
+		url('../static/fonts/Druk-Bold.ttf') format('truetype');
+	font-weight: normal;
+	font-style: normal;
+}
+
+// Post Grotesk
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-Medium.eot');
+	src: url('../static/fonts/PostGrotesk-Medium.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-Medium.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-Medium.ttf') format('truetype');
+	font-weight: bold;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-Regular.eot');
+	src: url('../static/fonts/PostGrotesk-Regular.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-Regular.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-Regular.ttf') format('truetype');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-BookItalic.eot');
+	src: url('../static/fonts/PostGrotesk-BookItalic.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-BookItalic.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-BookItalic.ttf') format('truetype');
+	font-weight: normal;
+	font-style: italic;
+}
+
+@font-face {
+	font-family: 'Post Grotesk';
+	src: url('../static/fonts/PostGrotesk-Light.eot');
+	src: url('../static/fonts/PostGrotesk-Light.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/PostGrotesk-Light.woff') format('woff'),
+		url('../static/fonts/PostGrotesk-Light.ttf') format('truetype');
+	font-weight: 300;
+	font-style: normal;
+}
+
+// Stanley
+
+@font-face {
+	font-family: 'Stanley';
+	src: url('../static/fonts/Stanley-Regular.eot');
+	src: url('../static/fonts/Stanley-Regular.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/Stanley-Regular.woff') format('woff'),
+		url('../static/fonts/Stanley-Regular.ttf') format('truetype');
+	font-weight: normal;
+	font-style: normal;
+}
+
+@font-face {
+	font-family: 'Stanley';
+	src: url('../static/fonts/Stanley-RegularItalic.eot');
+	src: url('../static/fonts/Stanley-RegularItalic.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/Stanley-RegularItalic.woff') format('woff'),
+		url('../static/fonts/Stanley-RegularItalic.ttf') format('truetype');
+	font-weight: normal;
+	font-style: italic;
+}
+
+// YoungSerif
+
+@font-face {
+	font-family: 'YoungSerif';
+	src: url('../static/fonts/YoungSerif-Regular.eot');
+	src: url('../static/fonts/YoungSerif-Regular.eot?#iefix') format('embedded-opentype'),
+		url('../static/fonts/YoungSerif-Regular.woff') format('woff'),
+		url('../static/fonts/YoungSerif-Regular.ttf') format('truetype');
+	font-weight: 600;
```

Comment:
Why not keep this normal? If there is only one weight will it not always need the `font-weight` value set when this typeface is used.